ui/frontend/src/components/RouteDetails.vue

```diff
@@ -235,6 +246,15 @@ export default {
       }
       this.formValues.plugins[propName] = v;
     },
+    updateInputFilesCollectionField(e) {
```

Comment:
it's copy of `updateCollectionField()`. Also why `!this.formValues` is required

Reply:
removed this check